test/system/280-update.bats

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Tests for podman update
+#
+
+load helpers
+
+LOOPDEVICE=
+
+function teardown() {
+    if [[ -n "$LOOPDEVICE" ]]; then
+        losetup -d $LOOPDEVICE
+        LOOPDEVICE=
+    fi
+    basic_teardown
+}
+
+
+@test "podman update - test all options" {
+
+    local cgv=1
+    if is_cgroupsv2; then
+        cgv=2;
+    fi
+
+    # Need a block device for blkio-weight-device testing
+    local pass_loop_device=
+    if ! is_rootless; then
+        if is_cgroupsv2; then
+            lofile=${PODMAN_TMPDIR}/disk.img
+            fallocate -l 1k  ${lofile}
+            LOOPDEVICE=$(losetup --show -f $lofile)
+            pass_loop_device="--device $LOOPDEVICE"
+
+            # Get maj:min (tr needed because losetup seems to use %2d)
+            lomajmin=$(losetup -l --noheadings --output MAJ:MIN $LOOPDEVICE | tr -d ' ')
+        fi
+    fi
+
+    # Shortcuts to make the table narrower
+    local -a gig=(0 1073741824 2147483648 3221225472)
+    local devicemax="1:5 rbps=10485760 wbps=31457280 riops=2000 wiops=4000"
+    local mm=memory/memory
+
+    # Format:
+    #   --<option> = <value>  | rootless? | check: cgroups v1            | check: cgroups v2
+    #
+    # Requires very wide window to read. Sorry.
+    #
+    # FIXMEs:
+    #  cpu-rt-period  (cgv1 only, on cpu/cpu.rt_period_us) works on RHEL8 but not on Ubuntu
+    #  cpu-rt-runtime (cgv1 only, on cpu/cpu.rt_runtime_us) fails: error setting cgroup config for procHooks ...
```

Comment:
UPDATE: #15666 sheds some light on why `cpu-rt-runtime` doesn't work. I don't have time right now to delve into it.

Reply:
Okay... as best I understand #15666, the `rt` options work only on some systems. I'm inclined to just change the above comment from `FIXMEs` to `Notes`, with maybe a link to that issue and an indication that those flags just aren't worth the hassle of testing.